app/controllers/workflows_controller.rb

```diff
@@ -22,7 +22,7 @@ def lifecycle
   def index
     # This ought to be a redirect, but our infrastructure is making it a
     # challenge to do right now: https://github.com/sul-dlss/workflow-server-rails/pull/162#issuecomment-479191283
-    # so we'll just ignore an log it.
+    # so we'll just ignore and log it.
     # return redirect_to "/objects/#{params[:druid]}/workflows" if params[:repo]
     logger.warn 'Workflows#index with repo parameter is deprecated. Call /objects/:druid/workflows instead' if params[:repo]
```

Comment:
@jcoyne can this go away yet?  Now that lyberservices-prod is no more?  If so, please do a quick PR or write a ticket?   Look also in config/routes.rb#L8-L18